### ISSUE TITLE
Escape from Tarkov

### ISSUE THREAD

**Issue by AuthFailed** [Starz0r/AreWeAntiCheatYet#269]
### Game Title

Escape from Tarkov

### Game Website

https://www.escapefromtarkov.com

### Anti-Cheat Software

BattlEye

### Status of the Anti-Cheat

❔ Unconfirmed

### Proof of Mention

N/A

**Comment by AuthFailed** [comment]
![2021-12-03_23-43](https://user-images.githubusercontent.com/21986465/144640030-988da835-d369-4f12-a84a-923c171b1338.png)

**Comment by AuthFailed** [comment]
Found in the discord "Tarkov Penguins" instructions on how to start the game and run offline raids.
https://discord.com/channels/891628783176650773/917612390969905202/917613274785275986
![изображение](https://user-images.githubusercontent.com/21986465/145522652-1eec6ead-c8a8-4e80-81a9-efe45ee3ac37.png)

**Comment by Vinjul1704** [comment]
Hey, not sure if this is trustworthy enough. I am the admin of the Discord server from the screenshot above and part of the Tarkov community team (mostly as a Discord mod, but I also help out with some testing and bug hunting).

I made a request to the developers and especially the companys boss, Nikita Buyanov, through the rest of the community team and we got the confirmation that they asked BattlEye for Linux support and that BattlEye plans to do it.

I got permission to share this information in public, although I'm not sure if the developers of the game plan to announce it in a more official matter through official news channels any time soon.

Attached is a screenshot of the response:
![image](https://user-images.githubusercontent.com/16061278/154179981-e68c3a0b-cf52-4b5a-a4ac-3a1ac01173c4.png)
